runtime/src/accounts_db.rs

```diff
@@ -1260,27 +1265,19 @@ impl AccountsDB {
         F: Fn(&StoredAccount, AppendVecId, &mut B) + Send + Sync,
```

Comment:
I think the above (out of the context diff) comemnt below isn't relevant or severely outdated anymore:

```
    // PERF: Sequentially read each storage entry in parallel
```